apps/wing/src/commands/upgrade.ts

```diff
@@ -1,26 +1,22 @@
 import debug from "debug";
+import { currentPackage } from "../util";
 
 const log = debug("wing:update");
-const PJSON = require("../../package.json");
 const DEFAULT_UPDATE_RATE = 1000 * 60 * 60 * 24; // 1 day
 
 /** Handles checking for toolchain updates */
 export async function checkForUpdates() {
-  if (!process.stdout.isTTY) {
-    log("not a TTY, skipping update check");
+  if (!process.stdout.isTTY || process.env.CI === "true" || currentPackage.version === "0.0.0") {
+    log("skipping update check");
     return;
   }
 
   log("checking for updates...");
-  // eval is because "update-notifier" is an ESM package and we're CommonJS.
-  const updateNotifier = await eval('import("update-notifier")');
-  const notifier = updateNotifier.default({
-    updateCheckInterval: DEFAULT_UPDATE_RATE,
-    pkg: PJSON,
+  // eval is because "tiny-updater" is an ESM package and we're CommonJS.
+  const updateNotifier = await eval('import("tiny-updater")');
```

Comment:
Not related to this PR but can't we use `await import()` even in CommonJS? 🤔

Reply:
Yessir, but tsc converts dynamic imports into a `require`-based promise when you are compiling to cjs. the eval bypasses that and forced us to use `import`

I can expand the comment to be more clear.

Reply:
If we run the cli through an actual bundler we probably don't need this, but I had some issues with that so I kept things mostly as-is

Reply:
Thanks for the info! Much better with that comment of top of it :)